.github/workflows/manual-clan-sync.yml

```
@@ -28,60 +22,73 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)